deployments/helm/k8s-dra-driver/templates/privilegedrolebinging.yaml

```diff
@@ -0,0 +1,17 @@
+# Apply only when running on OpenShift to let the kublet plugin run privileged
+{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
So you need both this new `RoleBinding` **_and_** the normal `ClusterRoleBinding` on openshift:
https://github.com/NVIDIA/k8s-dra-driver/blob/main/deployments/helm/k8s-dra-driver/templates/clusterrolebinding.yaml

Reply:
I tried to balance between POLP, flexibility, and the intrusiveness of changes here.  This solution seems good to me. E.g. the openshift stuff is kept separate and explicit. 
I've discarded my previous changes to the ClusterRole.

Reply:
My question was more about whether we should have the following in the normal `ClusterRoleBinding`:
```
{{- if not .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" -}}
...
{{- end }}
```
or if you guys actually need both on openshift.

Reply:
The `ClusterRoleBinding` was there already, I believe it's required for the normal functioning of the plugin on any K8s-based platform. The new `RoleBinding` is where OpenShift differs from the others.

Reply:
OK